### ISSUE TITLE
Chaturbate not working

### ISSUE THREAD

**Issue by scorsese50** [dobbelina/repository.dobbelina#777]
All category portals do not work ,When trying to open them Kodi just locks up with a busy screen and has to be force closed.

**Comment by frankee123** [comment]
Same. Weird how it's both chaturbate and MFC not working at same time.

**Comment by dobbelina** [comment]
Will be a fix for chaturbate in the next release https://github.com/dobbelina/repository.dobbelina/commit/b46b1a41dab735ff0394e030a64380c55cbde9ff

**Comment by TarkusLV** [comment]
The Chaturbate fix removes the HD, HD+, etc., labels.  Can those be added back, or are they gone for good?

**Comment by Gujal00** [comment]
@TarkusLV check the website, they are not providing those labels anymore

**Comment by TarkusLV** [comment]
> @TarkusLV check the website, they are not providing those labels anymore

Yeah, I see that.  Hopefully, they bring them back at some point, even if they weren't always accurate.

Thanks for your reply.
